api/institutions/views.py

```diff
@@ -231,6 +231,7 @@ class InstitutionRegistrationList(InstitutionNodeList):
     ordering = ('-date_modified', )
 
     def get_queryset(self):
+        inst = self.get_institution()
         query = self.get_query_from_request()
-        nodes = list(Node.find(query))
+        nodes = list(Node.find_by_institution(inst, query))
```

Comment:
This shouldn't need to be cast to a list since, you're creating a list on the next line anyway. This can be fixed in a follow up PR